org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/batch/messages.properties

```diff
@@ -268,12 +264,10 @@ misc.usage = {1} {2}\n\
 \    -20 -20.0          use 20  compliance (-source 20  -target 20)\n\
 \    -21 -21.0          use 21  compliance (-source 21  -target 21)\n\
 \    -22 -22.0          use 22  compliance (-source 22  -target 22)\n\
-\    -source <version>  set source level: 1.3 to 1.9, 10 to 22\n\
-\                       (or 6, 6.0, etc)\n\
-\    -target <version>  set classfile target: 1.3 to 1.9, 10 to 22\n\
-\                       (or 6, 6.0, etc)\n\
-\                       cldc1.1 can also be used to generate the StackMap\n\
-\                       attribute\n\
+\    -source <version>  set source level: 1.8, 9 to 22\n\
+\                       (or 8.0, 9.0, etc)\n\
+\    -target <version>  set classfile target: 1.8, 9 to 22\n\
+\                       (or 8.0, 9.0, etc)\n\
```

Comment:
Note, the formats were also missing in original version: `1.3 to 1.9, 10 to 22`.
I personally found it would make sense to show most common variants of versions. 
`1.8` is more common as `8` and till now I never saw `1.9`.

Reply:
This is how I read the original: 
* 1.3 to 1.9, 10 to 22
  * lists all versions in a consistent set of formats
*  (or 6, 6.0, etc)
   * show additional formats, that may be used alternatively to the above

This can be expanded to
* 1.3, 1.4, 1.5, 1.6, 6, 6.0, 1.7, 7, 7.0, 1.8, 8, 8.0, 1.9, 9, 9.0, 10, 10.0, 11, 11.0, 12, 12.0 .... 22, 22.0 

First goal would be to make the specification complete, second goal, make it nice, if that doesn't conflict with goal 1.

Reply:
OK, sounds good, will do.